Barotrauma/BarotraumaShared/SharedSource/Characters/Params/CharacterParams.cs

```diff
@@ -29,6 +29,15 @@ class CharacterParams : EditableParams
         [Serialize("", IsPropertySaveable.Yes, description: "If defined, different species of the same group consider each other friendly and do not attack each other."), Editable]
         public Identifier Group { get; private set; }
 
+        [Serialize("monster", IsPropertySaveable.Yes, description: "If changed, this character will try to play a custom music track when encountered."), Editable]
+        public string MusicType { get; private set; }
```

Comment:
This should be an `Identifier` instead of the `string`, because it's converted to an Identifier in all of the situations it's used in.